### PR TITLE
[issue-1051] German Localization

### PR DESCRIPTION
**Pull request by bloodscript**
This PR shall improve the german localization.

### CODE REVIEW

**Comment by bloodscript** [comment]
It is quite hard for a not native speaker to translate "you".
I see some translations using the polite form "Sie" or "Ihre", sometimes the familiar version "du" or "dein".
I think it would be nice to have this consistent.
Therefore my question:
Shall the more polite form "Sie" be used or the more commonly, especially for younger people used form "Du"?
Maybe this blogpost can help you understand the difference better and help with making a decision.
https://smartergerman.com/blog/sie-or-du/

My propose would be to use "Du".

**Comment by bloodscript** [comment]
@advplyr what would be your preferred solution?

**Comment by advplyr** [comment]
Whatever you think is appropriate. It may be context dependent in which case I can give you the context for any of the strings.
Most of the German translation has been done by @JBlond and @Hallo951

**Comment by bloodscript** [comment]
@JBlond and @Hallo951 could you please review my changes? I am a native speaker but for some strings I dont have the context yet. I can only test the iOS version and dont know how the usage of some strings "looks" maybe for some I would be appropriate to find shorter wordings.

**Comment by JBlond** [comment]
@bloodscript Your changes are fine. I like it. 

Some comments on the changes: 

- eBook and eReader might be like Duden suggests? [1][2] 
- "du" /"Deine(r/s)"  isn't consistent. Sometimes capitalized and some not. Please choose one form.


[1] https://www.duden.de/rechtschreibung/E_Book
[2] https://www.duden.de/rechtschreibung/E_Reader

**Comment by bloodscript** [comment]
Thank you @JBlond for the feedback.
I adapted eBook and eReader to E-Book and E-Reader (even though I personally would think eBook and eReader are nicer, as well as colleagues I asked).
I also now consistently use the uncapitalized versions of "du" or "dein" based on an blogpost of the Gesellschaft für deutsche Sprache e.V. (society for german language association) 
Link: https://gfds.de/du-dich-dein-dir-gross-oder-kleinschreibung-bei-den-anredepronomen/

**Review by JBlond — APPROVED** [review]

**Comment by Sapd** [comment]
Im also for using "Du".

Also some other translations in the Admin-Section are very weird. For example in OIDC: "Automatischer Start"

**Comment by advplyr** [comment]
Thanks!

**Comment by JBlond** [comment]
@bloodscript Please do a follow up for "du" vs "Sie" on https://github.com/advplyr/audiobookshelf/blob/master/client/strings/de.json  , the server version. That would be very kind.

**Comment by bloodscript** [comment]
Thanks for the hint I will have a look into it @JBlond
